spec/factories.rb

```diff
@@ -0,0 +1,21 @@
+FactoryBot.define do
+  factory :user do
+    name { Faker::TvShows::FamilyGuy.character }
+    email { Faker::Internet.email }
+    image { Faker::Internet.url }
+  end
+
+  factory :track do
+    track_path { Faker::Internet.url }
+    month { rand(13) }
+    year { 2021 }
+  end
+
+  factory :verse do
+    track_id { rand(100) }
+    user_id { rand(100) }
+    audio_path { Faker::Internet.url }
+    user
+    track
+  end
+end
```

Comment:
Love the implementation of factorybot

Reply:
Hahaha, thanks - looking at this now I think I need to specify a range for `month` in the track, I think that sometimes it going to return 0 which isn't a valid month